packages/shared/providers.yaml

```diff
@@ -198,6 +198,9 @@ discord:
     token_url: https://discord.com/api/oauth2/token
     authorization_params:
         response_type: code
+    proxy:
+        retry:
+            after: 'Retry-After'
```

Comment:
Btw. the header name for Discord is `X-RateLimit-Reset-After` ([docs](https://discord.com/developers/docs/topics/rate-limits#header-format))

Reply:
In this case I’m using the header that is sent once the rate limit has been hit 

> In the case that a rate limit is exceeded, the API will return a HTTP 429 response code with a JSON body. Your application should rely on the Retry-After header or retry_after field to determine when to retry the request.

Reply:
Oh, I see, sorry